javascript_nodejs/17.proactive-messages/dialogs/mainDialog/index.js

```diff
@@ -26,21 +30,22 @@ class MainDialog {
 
             // If user types in run, create a new job.
             if (utterance === "run"){
-                await this.createJob(this.storage, turnContext);
-            }
-
-            const firstWord = utterance.split(' ')[0];
-            const secondWord = utterance.split(' ')[1];
-
-            // If the user types done and a Job Id Number,
-            // we check if the second word input is a number.
-            if (firstWord === "done" && !isNaN(parseInt(secondWord))) {
-                var jobIdNumber = secondWord;
-                await this.completeJob(this.storage, turnContext, jobIdNumber);
-                await turnContext.sendActivity('Job completed. Notification sent.');
-
-            } else if (firstWord === "done" && isNaN(parseInt(secondWord))) {
-                await turnContext.sendActivity('Enter the job ID number after "done".');
+                await this.createJob(turnContext);
+            } else if (utterance === "show") {
+                await this.showJobs(turnContext);
+            } else {
+                const firstWord = utterance.split(' ')[0];
+                const secondWord = utterance.split(' ')[1];
```

Comment:
It would be nice to use array destructuring here, instead of two calls to utterance.split, but this is minor.

Reply:
I updated this.